Topics/User_Experience.md

```diff
@@ -25,3 +25,29 @@ ___Color Theory___
 ### Helpful Courses
 
 There are many courses related to User Experience. This includes the U of T Course [The Design of Interactive Computational Media](https://artsci.calendar.utoronto.ca/course/csc318h1) which is highly recommended if interested in this topic. [The Design of Interactive Computational Media](https://artsci.calendar.utoronto.ca/course/csc318h1) expands on the work done before coding projects. For example, one of the projects is testing how users would interact with a prototype of a UI and modifying it so that the UX is better for the user.
+
+### Tools Can Be Used In UX Design
+
+In addition to Bootstrap, Ant Design is another popular UI library and design system. It is specifically designed for React applications, providing a complete set of high-quality React components that make user interface development simpler, more consistent, and modular. Ant Design is ideal for building enterprise-level applications and dashboards with a focus on high performance, usability, and aesthetics.
+
+Ant Design provides a wide range of UI components, including buttons, icons, form elements, tables, and navigation menus. It also offers a consistent design language, making it easier for developers to create appealing user interfaces. The design ensures that your applications look great on different devices and screen sizes. More importantly, it is easy to use for beginners. Following steps are provided for you to start using Ant Design in your project.
+
+First, you need to install Ant Design in your React application. You can do this by running the following command:
+
+**npm install antd**
+
+Next, you may want to import the components you want to use in your application. For example, if you want to change the style for your button in a webpage, simply import Button from antd package.
+
+**import { Button } from 'antd';**
```

Comment:
note that when a developer wish to use ant's UI component, after importing the component, the developer also needs to change the tag in the code by checking its official document, such as <button> to <Button> . There are other component tags that are very different from the original html tag and by providing a basic but complete example would help new comers to get on board a lot easier